LDAP-Auth/Api/LdapController.cs

```diff
@@ -15,6 +20,17 @@ namespace Jellyfin.Plugin.LDAP_Auth.Api
     [Produces(MediaTypeNames.Application.Json)]
     public class LdapController : ControllerBase
     {
+        private readonly LdapAuthenticationProviderPlugin _ldapAuthenticationProvider;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LdapController"/> class.
+        /// </summary>
+        /// <param name="appHost">The application host to get the LDAP Authentication Provider from.</param>
+        public LdapController(IApplicationHost appHost)
+        {
+            _ldapAuthenticationProvider = appHost.GetExports<IAuthenticationProvider>().OfType<LdapAuthenticationProviderPlugin>().FirstOrDefault();
```

Comment:
```suggestion
            _ldapAuthenticationProvider = appHost.GetExports<LdapAuthenticationProviderPlugin>(false).First();
```

Might as well only create the provider we need